api/database/seeders/PoolCandidateTestSeeder.php

```diff
@@ -57,7 +67,12 @@ public function run()
             ->asApplicant()
             ->withSkillsAndExperiences()
             ->afterCreating(function (User $user) {
-                $this->applyToAllPools($user, PoolCandidateStatus::QUALIFIED_AVAILABLE, now()->addYears(2));
+                $this->applyToAllPools(
+                    user: $user,
+                    status: PoolCandidateStatus::QUALIFIED_AVAILABLE->name,
+                    expiryDate: now()->addYears(2),
+                    veteranStatus: ClaimVerificationResult::ACCEPTED->name,
```

Comment:
This should be unverified as per the document

Reply:
Correct! https://github.com/GCTC-NTGC/gc-digital-talent/pull/11617/commits/f40bf31b04277e0d5460bb5092b65b5a8e405b03